### PR TITLE
Switched getExtraProps to use findClosestFunction to get the component name.

### PR DESCRIPTION
**Pull request by mmcote**
### Summary of Changes
- Changed getExtraProps to use findClosestFunction, which has now been changed to findClosestofSymbolDeclaration to find the name of the component even when in production  code.

### Screenshots
Before
screen shot 2018-04-03 at 1 58 12 pm

After
screen shot 2018-04-03 at 2 02 36 pm

### CODE REVIEW

File: src/utils/ast.js [hunk 1]
```diff
@@ -68,11 +71,11 @@ export function containsPosition(a: AstLocation, b: AstPosition) {
   return startsBefore && endsAfter;
 }
 
-export function findClosestFunction(
-  functions: FunctionDeclaration[],
+function findClosestofSymbolDeclaration(
```
Comment (jasonLaster) [inline src/utils/ast.js]: nit: `findClosestofSymbol` is smaller :)

File: src/utils/ast.js [hunk 2]
```diff
@@ -100,3 +103,25 @@ export function findClosestFunction(
...
+}
+
+export function findClosestClass(
+  symbols: SymbolDeclarations,
+  location: Location
+) {
+  const { classes } = symbols;
+  if (!classes) {
+    return null;
+  }
```
Comment (jasonLaster) [inline src/utils/ast.js]: we can move this to `findClosestofSymbol`

**Comment by darkwing** [comment]
Wow, this looks slick @mmcote!  Nice work!

So I'm trying to test this on a React app (debugger.html) and I'm not seeing the `t` or `App` panes;  any idea what I could be doing wrong?

Thanks!

**Comment by jasonLaster** [comment]
We can look into the debugger case separately... This logic works well and the code reads well.

**Pull request merged**